ee_tests/config/local_osio.conf.sh.template

```diff
@@ -14,13 +14,11 @@ export GITHUB_PASSWORD="${GITHUB_PASSWORD:-}"
 #export GITHUB_REPO="${$GITHUB_REPO:-testmay-to-be-imported}"
 
 # test params
-export TEST_SUITE="${TEST_SUITE:-runTest}"
+# see protractor.config.ts for valid test suites
+export TEST_SUITE="${TEST_SUITE:-all}"
```

Comment:
I would probably avoid `all` because it runs also unmaintained tests and therefore it might fail. What we run most often is `smoketest` (that is a successor of `runTest`). The smallest & quickest test suite that runs regularly on Jenkins is `logintest`. I would choose one of those.